timeout_sampler/__init__.py

```diff
@@ -62,7 +61,7 @@ class BExampleError(AExampleError)
         sleep (int): Time in seconds between calls to func
         func (Callable): to be wrapped by TimeoutSampler
         exceptions_dict (dict): Exception handling definition
-        print_log (bool): Print elapsed time to log
+        print_log (bool): Print elapsed time to log. Deprecated
```

Comment:
Are you planning to make the print_log always be true in future?

Reply:
removed.